service-loadbalancer/service_loadbalancer.go

```diff
@@ -277,6 +301,13 @@ func (cfg *loadBalancerConfig) write(services map[string][]service, dryRun bool)
 	conf["startSyslog"] = strconv.FormatBool(cfg.startSyslog)
 	conf["services"] = services
 
+	conf["sslCert"] = ""
+	if cfg.sslCert != "" {
+		conf["sslCert"] = "crt " + cfg.sslCert
+	}
+	if cfg.sslCaCert != "" {
+		conf["sslCert"] = conf["sslCert"].(string) + " ca-file " + cfg.sslCaCert
```

Comment:
I've tested this on my kubernetes cluster and it works. So I think it is ready to merge for a first version. Its a little bit painful to develop for kubernetes cluster on a windows system. So it takes a lot of time for me, sorry.